### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by phillprice**
Is this the correct logic?

master/1/a.txt
master/2/b.txt

slave/2/b.txt
slave/3/c.txt

sync happens here

master/1/a.txt
master/2/b.txt

slave/1/a.txt
slave/2/b.txt

1 is copied from master to slave
2 is the same on both
3 folder on slave is deleted ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ It helps you sync 2 directories at a time. There are two types.
 
 Master
 ------
-Contents of this directory are moved to Slave for writing and updating. If this folder has a path that Slave doesn't then the path on Slave is deleted.
+Contents of this directory are moved to Slave for writing and updating. If Slave has a folder has a path that this doesn't then the path on Slave is deleted.
```
Comment (danhunsaker) [inline README.md:18]: More correctly, grammar-wise, "If Slave has a path that this folder doesn't, then"

**Comment by danhunsaker** [comment]
While we're updating the README anyway, it might be useful to mention that the deletes are optional.  I know it's mentioned later, when it lists the three phases and how to trigger them, but I suspect some users would stop reading after seeing things are being deleted on Slave, if that's not a behavior they're looking for.  Especially if they're in a hurry, which many developers are while looking for new packages, in my experience.

Whether we mention the sync is split across three phases (create, update, delete), or simply change "is deleted" to "is (optionally) deleted", or something else entirely, is up for discussion (or left to personal taste of whoever supplies the edit).  I just see it as a useful change for attracting new users.

File: README.md (hunk 1 above)
Comment (thadbryson) [reply README.md:18]: That logic is correct Phil. And Dan that wording is a little off. I'm going to leave that wording on the deleting alone. Maybe something like "If Slave has a folder that this directory doesn't: then it is deleted." What do you guys think?

**Review by thadbryson — CHANGES_REQUESTED** [review]
Can you change the wording to something like my comment?

**Pull request closed without merging**